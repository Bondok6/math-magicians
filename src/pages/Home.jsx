import React from 'react';

const Home = () => <div>Hello Home</div>;

export default Home;
