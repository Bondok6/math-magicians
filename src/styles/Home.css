.home-section {
  margin-top: 5rem;
}

.home-section h2 {
  font-size: 3rem;
}

.home-section p {
  margin: 4rem 0;
  text-align: justify;
}
