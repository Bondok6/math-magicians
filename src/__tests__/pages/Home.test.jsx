import Home from './../../pages/Home';
import { render } from '@testing-library/react';

it('test home component', async () => {
  const tree = render(<Home />);
  expect(tree).toMatchSnapshot();
});
