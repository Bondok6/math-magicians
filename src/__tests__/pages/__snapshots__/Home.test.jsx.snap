// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test home component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class=" container home-section"
      >
        <h2>
          Welcome to our page!
        </h2>
        <p>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quas eius, dolorum ipsum consequatur nostrum eaque aut repellat id in non ut numquam? Nobis beatae mollitia sapiente esse perferendis libero nulla. Lorem ipsum dolor sit, amet consectetur adipisicing elit. Accusantium reiciendis molestias, earum a ab cumque totam! Minima praesentium deserunt earum, doloribus sequi modi ratione alias eum voluptates similique blanditiis vel!
        </p>
        <p>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsam excepturi alias, tempora id possimus blanditiis laudantium laboriosam inventore nam explicabo hic placeat eum, voluptatem amet aliquid mollitia molestiae cum? Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quae nam nobis fugiat tempore tempora, nulla explicabo fuga perferendis eveniet esse molestiae earum. Non accusantium unde, veniam sit deleniti quis ut.
        </p>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class=" container home-section"
    >
      <h2>
        Welcome to our page!
      </h2>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Quas eius, dolorum ipsum consequatur nostrum eaque aut repellat id in non ut numquam? Nobis beatae mollitia sapiente esse perferendis libero nulla. Lorem ipsum dolor sit, amet consectetur adipisicing elit. Accusantium reiciendis molestias, earum a ab cumque totam! Minima praesentium deserunt earum, doloribus sequi modi ratione alias eum voluptates similique blanditiis vel!
      </p>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsam excepturi alias, tempora id possimus blanditiis laudantium laboriosam inventore nam explicabo hic placeat eum, voluptatem amet aliquid mollitia molestiae cum? Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quae nam nobis fugiat tempore tempora, nulla explicabo fuga perferendis eveniet esse molestiae earum. Non accusantium unde, veniam sit deleniti quis ut.
      </p>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
